docs/how-tos/dbt_extensions.rst

```diff
@@ -80,6 +89,9 @@ Step 4. Add your custom transforms
 Here is where you will need an understanding of dbt, Clickhouse, Aspects' data schemas, and the specific transforms you
 want to create.
 
+If you need any python dependencies beyond what is provided by aspects-dbt, add these to a ``requirements.txt`` file at
+the top level of your repository.
+
```

Comment:
Note: [this change](https://github.com/openedx/tutor-contrib-aspects/pull/877/files#diff-ab9493272ed9d2c60f468bc12868616e669474e31d0bf99c168bbb855be046b5R26-R32) makes the `requirements.txt` file optional.